charts/appdaemon/values.yaml

```diff
@@ -0,0 +1,86 @@
+# Default values for Appdaemon.
+
+image:
+  repository: acockburn/appdaemon
+  pullPolicy: IfNotPresent
+  tag: 4.0.5
+
+strategy:
+  type: Recreate
+
+service:
+  port:
+    port: 5050
+  # # Boilerplate config for code-server functionality, be sure to make changes to suit your environment
+  # additionalServices:
+  # - enabled: true
+  #   nameSuffix: vscode
+  #   type: ClusterIP
+  #   port:
+  #     port: 12321
+  #     name: vscode
+  #     protocol: TCP
+  #     targetPort: 12321
+
+env: {}
+  # HA_URL:
+  # DASH_URL:
```

Comment:
Should these be quoted in the values.yaml? Do you include http/https? Maybe add an example in here?

Reply:
I'll add a link that refers to the documentation over at appdaemons docs

Reply:
I like that, but the general question is more of a helm formatting thing. It's always nice to see an example you can edit.
```
env:
  #  HA_URL: "http://home.mydomain.com:8123"
  #  DASH_URL: http://home.mydomain.com:5050
```
Those are two ways to do it that people could use. Would both quoted and non work? My point here is that if we fill in with even one example people can follow that and not have to guess or go to the helm docs/google.

Reply:
I added a link to their docs, I don't think we need to include examples. We shouldn't need to add much documentation and have people refer to the upstream docs.

Reply:
I agree with @onedr0p . As a rule of thumb we try to avoid pre-filling container configuration if it can be avoided. 
This (hopefully 🙂) stimulates chart users to look at the upstream docs so they can gain an understanding of the settings they wish to modify, instead of changing an arbitrary value that we are providing as a default.